aip/general/0129.md

```diff
@@ -0,0 +1,150 @@
+---
+id: 129
+state: draft
+created: 2023-06-21
+placement:
+  category: resource-design
+  order: 65
+---
+
+# Server-Modified Values and Defaults
+
+Server-modified and default values often result in a poor user experience for
+[state-driven clients](state-driven-clients). These clients are often unable to
+tell when their desired state matches the current state for these fields, as the
+rules by which a server may modify and return values are complex, not public,
+and not repeatable. This AIP aims to clarify what behaviors can be expected from
+services and describing how they can be repeated on the client-side.
```

Comment:
Can this section just give context and not rationale? For example, this section could say something like:
> Services often provide default values for resource fields, and occasionally normalize the user input before returning it in the response. The guidance herein describes how services document such behavior for the benefit of consumers.

Then the [Rationale section](https://google.aip.dev/8#document-structure) discusses the why, for example:
> Server-modified and default values make it harder to implement and use state-driven clients. These clients are...

This is great reasoning, and it is exactly why we added the Rationale section.

Reply:
Added to base Rationale section, does that look right?